packages/react-icons/src/createIcon.tsx

```diff
@@ -7,18 +7,18 @@ export enum IconSize {
   xl = 'xl'
 }
 
-export const getSize = (size: IconSize | keyof typeof IconSize) => {
+export const getSizeClass = (size: IconSize | keyof typeof IconSize) => {
   switch (size) {
     case IconSize.sm:
-      return '1em';
+      return 'pf-svg-size-sm';
     case IconSize.md:
-      return '1.5em';
+      return 'pf-svg-size-md';
     case IconSize.lg:
-      return '2em';
+      return 'pf-svg-size-lg';
     case IconSize.xl:
-      return '3em';
+      return 'pf-svg-size-xl';
     default:
-      return '1em';
+      return 'pf-svg-size-sm';
   }
 };
```

Comment:
I can't find the definition for these classes. Did you mean to add them to [base/_common.scss](https://github.com/patternfly/patternfly/blob/main/src/patternfly/base/_common.scss)?

Reply:
@redallen we have an open core PR to add the classes - https://github.com/patternfly/patternfly/pull/3871